We don't care about warnings, we just want it to compile and work
Needed to compile on 32-bit arches

diff --git a/configure.ac b/configure.ac
index ff15596..d502682 100644
--- a/configure.ac
+++ b/configure.ac
@@ -185,7 +185,7 @@ fi
 
 AC_SUBST(CPP_GENFLAGS, ["-D_GNU_SOURCE -D_FILE_OFFSET_BITS=64"])
 
-AC_SUBST(C_GENFLAGS, ["-pthread -g -std=c99 -Wall -Wwrite-strings -Wmissing-prototypes -Werror -Wshadow"])
+AC_SUBST(C_GENFLAGS, ["-pthread -g -std=c99 -Wall -Wwrite-strings -Wmissing-prototypes -Wshadow"])
 AC_SUBST(C_OPTFLAGS, ["-O2"])
 AC_SUBST(C_DBGFLAGS, ["-g -DDEBUG"])
 
